src/main/java/com/thevoxelbox/voxelsniper/voxelsniper/Version.java

```diff
@@ -0,0 +1,13 @@
+package com.thevoxelbox.voxelsniper.voxelsniper;
+
+public enum Version {
+    V1_16,
+    V1_17,
+    V1_18,
+    V1_19;
+
+    //todo: Test if this works
+    public boolean isSupported(Version v) {
+        return Integer.parseInt(this.toString().substring(3)) >= Integer.parseInt(v.toString().substring(3));
```

Comment:
I would use this.name() instead, since toString can be overwritten, and name() can't.
```suggestion
        return Integer.parseInt(this.name().substring(3)) >= Integer.parseInt(v.name().substring(3));
```